base/applications/mspaint/CMakeLists.txt

```diff
@@ -1,5 +1,7 @@
 project(MSPAINT)
 
+remove_definitions(-D_WIN32_WINNT=0x502)
+add_definitions(-D_WIN32_WINNT=0x600) # SPI_GETWHEELSCROLLCHARS needs this
```

Comment:
What will it do when run on Win2003?
Will it still run but use a default value or will it need a newer Windows version?
If it's the former, that should be fine. The latter sounds a bit problematic for such a trivial feature.

Reply:
I know ReactOS supports `SPI_GETWHEELSCROLLCHARS`. But, this macro isn't defined in PSDK of `_WIN32_WINNT=0x502`.

Reply:
I can still use a preprocessor magic...

Reply:
Done.

Reply:
And you could always fall-back to e.g. code like the scrolling code in our console, that does not use this SPI setting.